### PR TITLE
first draft of ownCloud Web doc

### PR DESCRIPTION
**Pull request by EParzefall**
still a rough draft to be further developed

### CODE REVIEW

File: modules/owncloud_web/pages/index.adoc [hunk 1]
```diff
@@ -1,10 +1,41 @@
...
+If you are still running ownCloud 10.x as server, consider installing the {oc-marketplace-url}/apps/web[Web] app, which provides the new look and feel plus some additional features, but not everything ownCloud Web has to offer when used with oCIS. For a quick introduction about what's new or different, check out the section xref:web_with_oC10.adoc[ownCloud Web on ownCloud Server 10.x].
 
 // Test for oCIS
 
 // [source,yaml]
 // ----
 // include::https://raw.githubusercontent.com/wkloucek/cst-workshop/main/workshop-2022-02-02-docker/solution/docker-compose.yml[]
 // ----
+
+// I guess the above can be deleted now?
```
Comment (mmattel) [inline modules/owncloud_web/pages/index.adoc:22]: agreed

File: modules/owncloud_web/pages/index.adoc [hunk 2]
```diff
@@ -1,10 +1,41 @@
...
+
+== Changes
+
+For readers already familiar with ownCloud 10 and its standard web interface, the big differences in the new ownCloud Web on oCIS are:
+
+* A different sharing concept has been implemented,
+
```
Comment (mmattel) [inline modules/owncloud_web/pages/index.adoc:29]: linebreaks in list items are not necessary, but you can keep them if you desperately want them 😅

File: modules/owncloud_web/pages/web_for_admins.adoc [hunk 3]
```diff
@@ -0,0 +1,11 @@
+= ownCloud Web for Admins
+
+:toc: right
+:toc-levels: 1
+
+== User Management
+
+== App Integration
+
+Applications like Microsoft Word run in a "frame" displayed on oCIS, meaning they are actually running on the host server in this example at Microsoft, not on oCIS. Alternatively you can install the respective apps from the ownCloud marketplace.
```
Comment (mmattel) [inline modules/owncloud_web/pages/web_for_admins.adoc:10]: I think we have to rephrase this a bit to:
* better distinguish between ocis and oc10
* when do we install an app and when do we use an iframe displaying a foreign running application etc

File: modules/owncloud_web/pages/web_with_oC10.adoc [hunk 4]
```diff
@@ -0,0 +1,35 @@
+= ownCloud Web on ownCloud Server 10.x
+
+To ease the transition from ownCloud server 10.x to oCIS, we recommend starting to use the new Web UI. ownCloud users will benefit from the following additions and improvements:
```
Comment (mmattel) [inline modules/owncloud_web/pages/web_with_oC10.adoc:3]: have a `:description:` and introduction item here too. This should always be present right from the start in every document.

**Review by mmattel — APPROVED** [review]
Minor corrections necessary, approving to ease merging when done

File: modules/owncloud_web/pages/index.adoc (hunk 1 above)
Comment (EParzefall) [inline modules/owncloud_web/pages/index.adoc:22]:
```suggestion

```

File: modules/owncloud_web/pages/index.adoc (hunk 2 above)
Comment (EParzefall) [reply modules/owncloud_web/pages/index.adoc:29]: > linebreaks in list items are not necessary, but you can keep them if you desperately want them sweat_smile

Let's weed them out with the next revision.

File: modules/owncloud_web/pages/web_for_admins.adoc (hunk 3 above)
Comment (EParzefall) [inline modules/owncloud_web/pages/web_for_admins.adoc:10]:
```suggestion
// Applications like Microsoft Word run in a "frame" displayed on oCIS, meaning they are actually running on the host server in this example at Microsoft, not on oCIS. Alternatively you can install the respective apps from the ownCloud marketplace.
// needs more background
```

File: modules/owncloud_web/pages/web_with_oC10.adoc [hunk 5]
```diff
@@ -0,0 +1,35 @@
+= ownCloud Web on ownCloud Server 10.x
+
```
Comment (EParzefall) [inline modules/owncloud_web/pages/web_with_oC10.adoc:2]:
```suggestion
:description:
ownCloud web is also available for ownCloud server 10.x as an app to get familiar with the new interface.
```

File: modules/owncloud_web/pages/web_with_oC10.adoc (hunk 4 above)
Comment (EParzefall) [inline modules/owncloud_web/pages/web_with_oC10.adoc:3]:
```suggestion
== Introduction
{description}

To ease the transition from ownCloud server 10.x to oCIS, we recommend starting to use the new Web UI. ownCloud users will benefit from the following additions and improvements:
```

**Pull request merged**